packages/medusa-plugin-sentry-ui/README.md

```diff
@@ -21,21 +21,21 @@
 
```

Comment:
suggestion: Could you add a link to the documentation website to the right location please

Reply:
Oh yea, sure